client/electron/windows/OutlineService/OutlineService/OutlineService.cs

```diff
@@ -103,7 +103,14 @@ public partial class OutlineService : ServiceBase
         private EventLog eventLog;
         private NamedPipeServerStream pipe;
         private string proxyIp;
+
+        // Next-hop of the network adapter connected to the internet
         private string gatewayIp;
+
+        // IPv4 of the network adapter connected to the internet
+        private string gatewayAdapterIp;
```

Comment:
What's the difference between `gatewayIP` and `gatewayAdapterIP`?

What's the difference between `gatewayAdapter` and `gatewayInterface`?

We should be consistent with the naming.

Reply:
Removed.